src/Resources/app/administration/src/module/sw-dashboard/component/sw-dashboard-statistics-promotion-card/sw-dashboard-statistics-promotion-card.html.twig

```diff
@@ -0,0 +1,24 @@
+<sw-card
+    v-if="showBanner"
+    class="sw-dashboard-statistics-promotion-card"
+    >
+    <div class="sw-dashboard-statistics-promotion-card__decoration-canvas"></div>
+
+    <div class="sw-dashboard-statistics-promotion-card__content">
+        <h3 class="sw-dashboard-statistics-promotion-card__title">
+            {{ $tc('title') }}
+        </h3>
+
+        <div class="sw-dashboard-statistics-promotion-card__promotion-text">
+            {{ $tc('promotion-text') }}
+        </div>
+
+        <sw-button
+            variant="primary"
+            :disabled="!linkToStatisticsAppExists"
+            @click="goToStatisticsAppDetailPage"
```

Comment:
You can use the `router-link` prop here:

```suggestion
            :router-link="{ name: 'sw.extension.store.detail', params: { id: this.extension.id } }"
```

Reply:
If I add this to the button, it will be rendered as an anchor <a></a>. Then even if I set the disabled property, I can still click it (even though the cursor will look like you cannot).
It's not a big problem because it should never occur that the button is disabled (it's set to disabled when we cannot call the SBP to get the extension information, because we need the ID to create the link with the router), but it's still a bit weird.
I kept the button and onClick approach for now. Wdyt?

Reply:
This is definitively a bug in the component library then.  The button should be disabled. But if this is not possible we can stick to the onClick approach